modules/update-upgrading-cli.adoc

```diff
@@ -58,6 +58,23 @@ $ oc get clusterversion -o json|jq ".items[0].spec"
 For production clusters, you must subscribe to a `stable-\*` or `fast-*` channel.
 ====
 
+. Optional: To change the default `upstream` parameter value in cluster version run:
++
+[source,terminal]
+----
+$ oc edit clusterversion -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+  ...
+  spec:
+    clusterID: aba9371b-2f1b-4f3e-91a4-842824d829aa
+    upstream: https://api.openshift.com/api/upgrades_info/v1/graph
```

Comment:
If taking my last suggestion, the upstream needs to be updated to `https://example.com/graph` accordingly.

Reply:
ack